Sources/SimpleAnalytics/RequestDispatcher.swift

```diff
@@ -6,50 +6,19 @@
 //
 
 import Foundation
-import Alamofire
 
 internal struct RequestDispatcher {
     /// Sends the event to Simple Analytics
     /// - Parameter event: the event to dispatch
     static internal func sendEventRequest(event: Event) async throws {
-        return try await withCheckedThrowingContinuation { continuation in
-            AF.request("https://queue.simpleanalyticscdn.com/events",
-                       method: .post,
-                       parameters: event,
-                       encoder: JSONParameterEncoder.default).responseData { response in
-                
-                switch(response.result) {
-                    case .success(_):
-                        continuation.resume()
-                    case let .failure(error):
-                        continuation.resume(throwing: self.handleError(error: error))
-                    }
-            }
-        }
+        guard let url = URL(string: "https://queue.simpleanalyticscdn.com/events") else { throw URLError(.badURL) }
+        var request = URLRequest(url: url)
+        request.httpMethod = "POST"
+        request.setValue("application/json", forHTTPHeaderField: "Content-Type")
+        let jsonData = try JSONEncoder().encode(event)
+        request.httpBody = jsonData
+        let (_, response) = try await URLSession.shared.data(for: request)
+        guard let httpResponse = response as? HTTPURLResponse else { throw URLError(.badServerResponse) }
+        guard (200...299).contains(httpResponse.statusCode) else { throw URLError(URLError.Code(rawValue: httpResponse.statusCode)) }
     }
-    
-    static private func handleError(error: AFError) -> Error {
```

Comment:
no longer needed if we just handle with the sendEventRequest with vanilla URLSession